### ISSUE TITLE
Opening file in app running in Crossover in Distrobox doesn't work

### ISSUE THREAD

**Issue by eylenburg** [89luca89/distrobox#1174]
**Describe the bug**
I have installed Crossover in a Distrobox. Crossover is an application to run Windows apps, based on Wine. I have exported the Windows apps using `distrobox-export`. For example `distrobox-export --app 'Excel 2016'` resulted in a .desktop file like this: `/usr/bin/distrobox-enter -n debian -- /bin/sh -l -c /home/user/.cxoffice/Microsoft_Office_2016/desktopdata/cxmenu/StartMenu.C^5E3A_ProgramData_Microsoft_Windows_Start^2BMenu/Programs/Excel+2016.lnk %u`

I can use this to start (in this example) Excel 2016 running on Crossover running in a Distrobox. But if I want to open a file like this (right-click -> open with -> select that starter) it just opens Excel with a blank spreadsheet rather than opening the file.

**To Reproduce**
Right-click on file -> open with -> select the starter created by distrobox-export. It will start the app but won't open the file in the app.

**Expected behavior**
The selected file opens in the (here) Excel.

**Desktop (please complete the following information):**
 - Are you using podman, docker or lilipod? podman
 - Which version or podman, docker or lilipod? 4.8.3
 - Which version of distrobox? 1.6.0
 - Which host distribution? openSUSE
 - How did you install distrobox? from repo

**Additional context**
I previously had Crossover installed on the host directly and it was possible to open a file with a Crossover app through the "open with" dialogue. Because Crossover doesn't fully support openSUSE and some apps weren't working so well, I decided to install it in a Debian container. Hence I'm filing the bug report here, because this problem only occurs with Distrobox.

Given the fact that the .desktop file contains various commands and names including spaces, I have a feeling that it's just something silly like some quotation marks missing somewhere. Nevertheless this should work by default when using distrobox-export.

**Comment by wuyougan** [comment]
I have encountered the same situation. The executed sh command should be enclosed in double quotes, for example:

```
/usr/bin/distrobox-enter -n debian -- /bin/sh -l -c "/home/user/.cxoffice/Microsoft_Office_2016/desktopdata/cxmenu/StartMenu.C^5E3A_ProgramData_Microsoft_Windows_Start^2BMenu/Programs/Excel+2016.lnk %u"
```

**Comment by 89luca89** [comment]
Hi @eylenburg 

this is fixed now thanks to commit 3b1e312f
